tests/integration/targets/mso_schema_site_bd_subnet/tasks/main.yml

```diff
@@ -521,7 +588,7 @@
 - name: Verify non_existing_template_bd
   assert:
     that:
-    - non_existing_template_bd.msg == "Provided BD 'non_existing_template_bd' does not exist. Existing template BDs{{':'}} ansible_test_1"
+    - non_existing_template_bd.msg == "Provided condition(s) ['name==non_existing_template_bd'] not matching existing object(s){{':'}} ['name==ansible_test_create_1'], ['name==ansible_test_1']"
```

Comment:
This error message is not really user friendly. Why changing it from what it was before?

Reply:
changed because I tried to create a generic message from within the schema class, we can change this message structure

Reply:
changed as discussed